JS/app.js

```diff
@@ -0,0 +1,81 @@
+var allProducts = [];
+var productName = ['bag', 'banana', 'boots', 'chair', 'cthulhu', 'dragon', 'pen', 'scissors', 'shark', 'sweep', 'unicorn', 'usb', 'water_can', 'wine_glass'];
+
+function Product(name) {
+  this.name = name;
+  this.voteCount = 0;
+  this.path = 'images/' + name + '.jpg';
+  allProducts.push(this);
+};
+
+for (product in productName) {
+  var newProduct = new Product(productName[product]);
+}
+
+var productRanker = {
+  imageSec: document.getElementById('images'),
+  img1: document.getElementById('pic1'),
+  img2: document.getElementById('pic2'),
+  img3: document.getElementById('pic3'),
+  resultsList: document.getElementById('resultList'),
+  buttResults: document.getElementById('results'),
+  buttReset: document.getElementById('reset'),
+  timeRun: 0,
```

Comment:
For readability, you can toss an extra line break between methods of the object.